src/components/share-modal/share-modal-component.jsx

```diff
@@ -1,26 +1,24 @@
 import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { Modal, Button } from 'he-components';
-import { ReactComponent as ShareIcon } from 'icons/share.svg';
 import cx from 'classnames';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
-import ReactTooltip from 'react-tooltip';
 import { getShortenUrl } from 'services/bitly';
 
 import styles from './share-modal-styles.module';
 
 const LINK = 'link';
 const EMBED = 'embed';
 
-const ShareModal = ({ handleClose, isOpen, shareSocialMedia }) => {
+const ShareModalComponent = ({ handleClose, isOpen, shareSocialMedia }) => {
   const [activeTab, setActiveTab] = useState(LINK);
   const [copied, setCopied] = useState({ [LINK]: false, [EMBED]: false });
   const [shortLink, setShortLink] = useState(null);
   const [shareUrl, setShareUrl] = useState(null);
 
   const isActiveTabLink = activeTab === LINK;
   const currentLocation = window.location.href;
-  
+
```

Comment:
Sorry for bringing this here, not related to this PR, but we need to create the share link only when the modal `isOpen`. By doing it that way we would get two needed things:
- Create the `bit.ly` link just when the user explicitly interacts with the share functionality (savin url creation from the free account :)
- Share the actual state of the app in some cases where the user updates it after the modal has been mounted (that happens for example when entering the national report cards and the user visits the challenges chart, for example, the share url was already created and when it is shared it points to the map instead to the chart).

Doing this would do the trick :point_down: 

```js
useEffect(() => {
    if (isOpen) {
      getShortenUrl(currentLocation)
      .then(link => {
        setShortLink(link);
      })
    }
  }, [isOpen])
```